published/005-2016-Feb-1.md

```diff
@@ -0,0 +1,209 @@
+* [**distributions**](//github.com/ipfs/distributions)
+  - [dist.ipfs.io](http://dist.ipfs.io/) _almost_ has [signed releases](https://github.com/ipfs/distributions/pull/51) thanks to [@dignifiedquire](//github.com/dignifiedquire). This will be coming soon.
+	- [@noffle](//github.com/noffle) [worked](https://github.com/ipfs/distributions/pull/42) to get [ipget](http://ipget.me/) ready to be on [dist.ipfs.io](http://dist.ipfs.io/)
+* [**gx**](//github.com/whyrusleeping) 
+	- [@whyrusleeping](//github.com/whyrusleeping) shipped the GX PR, which pulls relevant code out of go-ipfs and into the new GX package manager. He also wrote a tool to publish IPNS records without the IPFS daemon.
+	- [@RichardLitt](//github.com/RichardLitt) spruced up the [gx README](https://github.com/whyrusleeping/gx) - it is almost ready to ship. 
+* [**js-ipfs**](//github.com/ipfs/js-ipfs) 
+	- Thanks to @diasdavid, `object features` now works, with tests and all. 
+	- As well, thanks to [@dignifiedquire](//github.com/dignifiedquire)'s efforts, the js-ipfs API now returns promises if there isn't a specified callback, allowing both major methods used by the Javascript community to work equally well.
+* [**registry-mirror**](//github.com/diasdavid/registry-mirror) 
+	- [@diasdavid](//github.com/diasdavid) merged registry-static into registry-mirror and removed all of the unnecessary stuff [(this was huge)](https://github.com/diasdavid/registry-mirror/pull/33).
+* [**station**](//github.com/ipfs/station)
+  - [@dignifiedquire](//github.com/dignifiedquire) fixed drag and drop
+    file uploading, and some dependency issues. So go ahead and try it
+    out.
+* [**ipfs-geoip**](//github.com/ipfs/ipfs-geoip)
+  - [@dignifiedquire](//github.com/dignifiedquire) rewrote the generation
+    script and cleaned up the code so that now the data is consistently
+    reproducable and fully stored on IPFS. This ensures that geoip lookups
+    over IFPS will work in all future releases.
+* [**fs-repo-migrations**](//github.com/ipfs/fs-repo-migrations)
+  - [@chriscool](//github.com/chriscool) Improved fs-repo-migrations tests by checking that the objects are the same and checking that garbage collection does not change anything. He also cleaned up sharness tests.
+* [**http-api-spec**](//github.com/ipfs/http-api-spec)
+  - @RichardLitt finished logging all existing `ipfs` commands for the HTTP API Spec. This means that if you have any questions about how the HTTP API should work, you can see them either in master or as open PRs to that issue. One step closer to a fully functional API...
```

Comment:
solid update. maybe make the call to action stronger: "If you are interested in how the HTTP API should work, or have any specific questions, please see the current version (link to it), and explore the open PRs to that issue." or something